src/screens/WalletConnectApprovalSheet.js

```diff
@@ -60,31 +60,22 @@ const DappLogo = styled(RequestVendorLogoIcon).attrs(
   marginBottom: 24,
 });
 
-const NetworkLabelText = styled(Text).attrs(({ theme: { colors } }) => ({
-  color: colors.alpha(colors.blueGreyDark, 0.6),
-  letterSpacing: 'roundedMedium',
-  size: 'smedium',
-  weight: 'semibold',
-}))({
-  marginBottom: 4,
-});
-
-const LabelText = styled(Text).attrs(({ theme: { colors } }) => ({
-  color: colors.dark,
-  size: 'large',
-  weight: 'heavy',
-}))({});
-
 const AvatarWrapper = styled(Column)({
   marginRight: 5,
   marginTop: 1,
 });
 
-const SwitchText = styled(Text).attrs(({ theme: { colors } }) => ({
-  color: colors.dark,
-  size: 'large',
-  weight: 'heavy',
-}))({});
+const LabelText = ({ children, ...props }) => (
```

Comment:
I would prefer to use curly brackets with return for such components because when you need to add anything else to ftap component (even console.log) - you have to rewrite it to curly brackets every time. so it's better to just default to curly brackets.

Reply:
Agree 🤝